rules/xcodeproj.bzl

```diff
@@ -81,11 +91,12 @@ def _xcodeproj_aspect_impl(target, ctx):
             actual = getattr(ctx.rule.attr, "tests")[0]
         elif ctx.rule.kind in ("alias"):
```

Comment:
On line 92, how does the `in` keyword work. The line is`ctx.rule.kind in ("alias"):`

Is `("alias")` treated as a list with one item?